sov-rollup-starter/src/register_sequencer.rs

```diff
@@ -0,0 +1,37 @@
+use anyhow::Context;
+use sov_modules_api::default_context::DefaultContext;
+use sov_modules_api::Spec;
+use sov_rollup_interface::da::DaSpec;
+use sov_rollup_interface::services::da::DaService;
+use sov_sequencer::batch_builder::FiFoStrictBatchBuilder;
+use sov_sequencer::get_sequencer_rpc;
+use sov_state::ProverStorage;
+use stf_starter::Runtime;
+
+/// register sequencer rpc methods.
+pub(crate) fn register_sequencer<Da>(
+    storage: &<DefaultContext as Spec>::Storage,
+    da_service: Da,
+    methods: &mut jsonrpsee::RpcModule<()>,
+) -> Result<(), anyhow::Error>
+where
+    Da: DaService,
+{
+    let batch_builder = create_batch_builder::<<Da as DaService>::Spec>(storage.clone());
```

Comment:
Just to give context, is it ok to clone here? Meaning: is the storage always expected to be `Arc`-like?

Reply:
Yes the storage is behind an `Arc`